modules/virt-about-deploying-templates-private-namespace.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-about-deploying-templates-private-namespace.adoc
+
+[id="virt-about-deploying-templates-private-namespace_{context}"]
+= About deploying templates to a private namespace
+
+To add templates to a private namespace for private use, edit the `hco` custom resource (CR) and specify a private namespace. Then, add the Red Hat templates to that private namespace.
+
+[NOTE]
+====
+Only you can view and use the templates deployed to your private namespace. Other users cannot view or use them.
```

Comment:
The word "you" here is not clear, I think it can be the project admin or users have edit permission of the project.

Reply:
Plan to make it clear by updating this content.

Reply:
First, what role can edit the HCO CR, I think it's only cluster admin can do that.
The word "you" in the note sounds like is the cluster admin now,  because he is the one who edit the HCO. So the note looks like only cluster admin can view and use the templates in private namespace. That's shouldn't be. Anyone who have view permission of the private namespace can view and use templates in that namespace.

Reply:
Updated the content.